### ISSUE TITLE
GPG-sign missing

### ISSUE THREAD

**Issue by tjunge** [magit/magit#3771]
A recent update to magit lost the option to GPG-sign commits.
  (1) Expected Behaviour
       Up until at least Magit 20190122.503, Git 2.20.1, Emacs 26.1, gnu/linux, when starting a commit, the following options were provided:
       Options
          =A Override the author (--author=)
          =S Sign using gpg (--gpg-sign="XXXXXXXXXXXXXX")
          =C Reuse commit message (--reuse-message=)

  (2) In version Magit 20190227.2130, Git 2.20.1, Emacs 26.1, gnu/linux, the Optons section has been dropped, and the option A and C have been grouped with the remaining arguments, but option S (gpg-sign) is no longer available.
  (3) To reproduce this issue, open the commit dialog in magit with a recent version of magit

**Comment by tarsius** [comment]
With the release of Transient, I have moved this argument to "level 5". See https://emacsair.me/2019/02/14/transient-0.1/ and https://magit.vc/manual/transient/Enabling-and-Disabling-Suffixes.html.

**Comment by mcraveiro** [comment]
I just bumped into this very same problem. As per @tarsius comment, I added the following to my magit setup:

```
(setq transient-default-level 5)
```

After this I was able to see the option again. HTH.

Marco

**Comment by gwood** [comment]
I am having trouble using the following line in ~/.emacs and solving the issue of using signatures again.

`(setq transient-default-level 5)`

I still get this error when I attempt to sign files:
```
128 git … commit --amend --only
hint: Waiting for your editor to close the file...
Waiting for Emacs...
error: cannot run gpg: No such file or directory
error: gpg failed to sign the data
fatal: failed to write commit object
```

Please let me know if I need to create a new issue.

I also asked this same question in [Emacs Stackexchange](https://emacs.stackexchange.com/questions/52720/magit-commit-does-not-permit-adding-a-gpg-signature).

### Versions of software and OS.
```
magit-version returns the following string

Magit 2.90.1 Git 2.20.1 (Apple Git-117), Emacs 26.1, darwin
on an iMac Pro (2017) running OS X 10.14.6

Version of Emacs: Emacs 26.1 (9.0)
https://www.gnu.org/software/emacs
```

**Comment by gwood** [comment]
I solved the above issue with this correction:

https://stackoverflow.com/questions/41741477/emacs-epa-and-gnupg2-no-usable-configuration

```
(setenv "PATH" (concat (getenv "PATH") ":/usr/local/bin"))
(setq exec-path (append exec-path '("/usr/local/bin")))

```
Then the next step was to choose an active KEY ID from the keyring.

In order to prove that I had a signature I needed to use a command line command:

git log --show-signature -1

From the command line.

I could not get Magit to display the signature
I tried to use the Magit Key (l) and the  -S switch.

But could not see the signature details.
